### PR TITLE
Boilerplate

### PR DESCRIPTION
**Pull request by justefg**

### CODE REVIEW

File: app.go [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    log.Info().Msg("Called /sign_transaction")
+    // TODO
+}
+
+func (app *App) InitializeRoutes() {
+    app.Router.HandleFunc("/ping", app.PingQuery).Methods("GET")
+    app.Router.HandleFunc("/sign_transaction", app.SignQuery).Methods("POST")
+}
+
+func getLevel(level string) zerolog.Level {
```
Comment (algys) [inline app.go]: move logger to another file, and mb make global ?
In future here will added logic related with events and its will represented in another file, so logger should be available for them too.

**Review by algys — CHANGES_REQUESTED** [review]

File: app.go (hunk 1 above)
Comment (justefg) [reply app.go]: it's already global

Comment (justefg) [reply app.go]: log.Logger is a global object

Comment (justefg) [reply app.go]: though I think moving getLevel and initLogger to a separate file is a good idea

**Review by algys — APPROVED** [review]

**Pull request merged**